components/form/LinkInverters.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import InverterGraphicIcon from '../icons/InverterGraphicIcon';
+import Link from 'next/link';
+import useMediaQuery from '~/lib/hooks/useMediaQuery';
+import Button from '../Button';
+import Spinner from '../Spinner';
+import { ChevronRightIcon } from '@heroicons/react/24/solid';
+
+const LinkInverters = () => {
+  const [showDisclaimer, setShowDisclaimer] = useState(false);
```

Comment:
tiny detail but I wouldn't call this `showDisclaimer`, since it's not really a disclaimer we're showing...
perhaps something like `showInfo` would be good here?